src/main/java/Duke.java

```diff
@@ -1,10 +1,158 @@
+        System.out.println("    ╔═══════════════════════════════════════════════════════════╗");
+        print(cmd);
+        System.out.println("    ╚═══════════════════════════════════════════════════════════╝");
+        System.out.println("    What can I do for you? Type 'bye' to exit.");
+        System.out.println();
+    }
+
+    public static void add(String cmd) {
+        Task todo = new Task(taskCounter, cmd, false);
+        tasks.add(todo);
+        taskCounter++;
+        System.out.println("    ╔═══════════════════════════════════════════════════════════╗");
+        print("Added: " + cmd);
+        System.out.println("    ╚═══════════════════════════════════════════════════════════╝");
+        System.out.println("    What can I do for you? Type 'bye' to exit.");
+        System.out.println();
+    }
+
+    private static void printTask(Task t) {
+        System.out.println("    ╔═══════════════════════════════════════════════════════════╗");
+        print("Got it. I've added this task: ");
+        print(t.toString());
+        print("Now you have " + taskCounter + " tasks in the list.");
+        System.out.println("    ╚═══════════════════════════════════════════════════════════╝");
+        System.out.println("    What can I do for you? Type 'bye' to exit.");
+        System.out.println();
+    }
+
+    public static void todo(String cmd) {
+        int startPosition = cmd.indexOf(" ");
+        String cmdTodo = cmd.substring(startPosition + 1, cmd.length());
+        Task todo = new Todo(taskCounter, cmdTodo, false);
+        tasks.add(todo);
+        taskCounter++;
+        printTask(todo);
+    }
+
+    public static void event(String cmd) {
+        int startPosition = cmd.indexOf(" ");
+        int timePosition = cmd.indexOf(" /at ");
+        String cmdEvent = cmd.substring(startPosition + 1, timePosition);
+        String cmdTime = cmd.substring(timePosition + charLengthToSkip, cmd.length());
+        Task event = new Event(taskCounter, cmdEvent, false, cmdTime);
+        tasks.add(event);
+        taskCounter++;
+        printTask(event);
+    }
+
+    public static void deadline(String cmd) {
+        int startPosition = cmd.indexOf(" ");
+        int timePosition = cmd.indexOf(" /by ");
+        String cmdDeadline = cmd.substring(startPosition + 1, timePosition);
+        String cmdTime = cmd.substring(timePosition + charLengthToSkip, cmd.length());
+        Task deadline = new Deadline(taskCounter, cmdDeadline, false, cmdTime);
+        tasks.add(deadline);
+        taskCounter++;
+        printTask(deadline);
+    }
+
+    public static void done(int taskNumber) {
+        tasks.get(taskNumber).setStatus();
+        System.out.println("    ╔═══════════════════════════════════════════════════════════╗");
+        print("Nice! I've marked this task as done: ");
+        print(tasks.get(taskNumber).getTaskType() + tasks.get(taskNumber).getStatusIcon() + " " + tasks.get(taskNumber).description);
```

Comment:
Consider putting this code in 2 lines instead of 1 (it is over 100 chars long). This is same for a few other overly long lines in your code.